esp-metadata/devices/esp32.toml

```diff
@@ -78,3 +79,7 @@ symbols = [
 ]
 
 memory = [{ name = "dram", start = 0x3FFA_E000, end = 0x4000_0000 }]
+
+[device.rmt]
```

Comment:
Is this data already available in a machine readable format? I can really see the appeal for this approach if we _don't_ have to manually specify this stuff.

Reply:
It would be better to encode this in the PACs, but we'd just move one set of handwritten constants to another place at this point.